packages/types/lib/driver.ts

```diff
@@ -469,7 +469,17 @@ export interface ISessionHandler<
    * @returns A session data object
    */
   getSession(): Promise<SingularSessionData<C, SessionData>>;
-}
+
+  /**
+   * Return session capabilities info.
+   * FIXME: Please define better type after deciding the response format in
+   * https://github.com/appium/appium/issues/20880
+   * @param sessionId - the id of the session to return capabilities info.
+   * @returns A session capabilities object
+   */
+  getAppiumSessionCapabilities?(sessionId: string): Promise<{
+    value: object, error?: Error, protocol?: string}>
```

Comment:
I believe this declaration should only return the type of the returned result. This protocol and error stuff is something that is specific for the umbrella implementation, but not for the base driver one